core/src/main/resources/schema/infinispan-config-11.0.xsd

```diff
@@ -1858,9 +1858,10 @@
         <xs:attribute name="path" type="xs:string">
           <xs:annotation>
             <xs:documentation>
-              Defines a location in the root path, specified in "relative-to",
-              where the cache state is stored. Defaults to the name of the
-              cache container.
+              Specifies a filesystem directory for data. The value should
+              always be relative to the global persistent location. If you set
```

Comment:
I think the use of the word "relative" is confusing here as it has different meanings when talking about paths:

https://en.wikipedia.org/wiki/Path_(computing)#Absolute_and_relative_paths

How about:

"Specifies a filesystem directory for data. The value can be a relative or absolute path. Relative paths are created relative to the configured global persistent location. Whereas absolute paths must be a subdirectory of the global persistent location, otherwise an exception is thrown."

Reply:
thanks, @ryanemerson that is spot on. tbh I felt I was missing some nuances in the wording. this is an excellent review!